src/js/me-i18n-locale-en.js

```diff
@@ -33,7 +33,7 @@
             'mejs.fullscreen' : 'Fullscreen',
 
             // mep-feature-jumpforward
-            'mejs.time-jump-forward': 'Jump forward %1 seconds',
+            'mejs.time-jump-forward': 'Jump forward {0} {0|plural:second:seconds}',
```

Comment:
What happens if a string or a translations has a semicolon? Shouldn't this be an array instead?